src/bin/rahmen.rs

```diff
@@ -170,7 +170,7 @@ fn main() -> RahmenResult<()> {
                 }
                 Configuration::ScreenDimensions(width, height) => Configuration::ScreenDimensions(
                     width,
-                    height - current_font_size.unwrap_or(0.) as u32,
+                    height - ( current_font_size.unwrap_or(0.) * 1.5) as u32, // this factor is the same as in dataflow.rs
```

Comment:
Please convert this to using the configuration parameter, similarly to `current_font_size`.